index.html

```diff
@@ -155,33 +174,32 @@ <h2>About me</h2>
                         <li>HTML</li>
                         <li>CSS</li>
                     </ul>
-                </li>
-                <li>
-                    <div class="footer-heading font">
-                        <img src="images/Rectangle.png" alt="Rectangle">
-                        <span class="font">Frameworks</span>
-                    </div>
-                    <ul class="inner-list">
-                        <li>React.js</li>
-                        <li>Ruby on Rails</li>
-                        <li>RSpec</li>
-                        <li>Capybara</li>
-                    </ul>
-                </li>
-                <li>
-                    <div class="footer-heading font">
-                        <img src="images/circle.png" alt="circle">
-                        <span class="font">Skills</span>
+                </div>
+                <div class="outer-list">
+                <div class="footer-heading font">
+                    <img src="images/Rectangle.png" alt="Rectangle">
+                    <span class="font">Frameworks</span>
+                </div>
+                <ul class="inner-list">
+                    <li>React.js</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSpec</li>
+                    <li>Capybara</li>
+                </ul>
+            </div>
+                    <div class="outer-list">
+                        <div class="footer-heading font">
+                            <img src="images/circle.png" alt="circle">
+                            <span class="font">Skills</span>
+                        </div>
+                        <ul class="inner-list">
+                            <li>Database Management</li>
+                            <li>Version Control</li>
+                            <li>CLI</li>
+                            <li>API Design</li>
+                            <li>Web development</li>
+                        </ul>
                     </div>
-                    <ul class="inner-list">
-                        <li>Database Management</li>
-                        <li>Version Control</li>
-                        <li>CLI</li>
-                        <li>API Design</li>
-                        <li>Web development</li>
-                    </ul>
-                </li>
-            </ul>
         </div>
     </section>
     <section class="contact-form">
```

Comment:
- [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Here, you should increase margin on the left side and `Get in touch` button should move to a little bit left to align with other text on the form and also give space under the social media icon. Kindly work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![contact_design](https://user-images.githubusercontent.com/70483835/229035777-ee343aab-1e26-4238-b06d-24519344ecdf.png) | ![contact_figma](https://user-images.githubusercontent.com/70483835/229035788-71ba255c-be6e-4758-bbb8-8c2fae0aa6da.png)